.github/workflows/openapi-check.yml

```
@@ -45,9 +45,30 @@ jobs:
       - name: Generate current OpenAPI spec
         run: |
           # Start server in background to fetch spec from API endpoint
-          MIX_ENV=test mix phx.server &
+          MIX_ENV=dev mix phx.server &
           SERVER_PID=$!
-          sleep 15  # Wait for server to start
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)